addresses.md

```diff
@@ -0,0 +1,54 @@
+# Grin Community Council Addresses <!-- omit in toc -->
+
+## Table of Contents <!-- omit in toc --> 
+- [About](#about)
+- [Verification](#verification)
+- [Contact](#contact)
+- [List of addresses](#list-of-addresses)
+  - [Grin General Fund](#grin-general-fund)
+    - [Grin](#grin)
+    - [Bitcoin](#bitcoin)
+      - [Legacy](#legacy)
+      - [Segwit](#segwit)
+    - [Ethereum](#ethereum)
+    - [Zcash](#zcash)
+
+## About
+This document is the only valid list of addresses for donations, sponsorships, or other forms of payments intended for the **Grin community council project**.
+For **GRIN Original Council** payment addresses see https://github.com/mimblewimble/grin-pm/blob/master/financials/addresses.md 
+
+## Verification
+This document should be hosted on the following address:
+
+**tbd**
+
+## Contact
+Any questions or concerns about authenticity can be raised publicly in the Grin community via [keybase](https://keybase.io/team/grincoin) 
+**tbd**
+
+---
+
+## List of addresses
+
+### Grin General Fund
```

Comment:
To me wording "general fund" is confusing with [grin general fund](https://grin.mw/fund) which (correct me if I'm wrong) is different than fund used by the Community Council.

Reply:
changed it to grin community council fund